tests/core/structure/cellReferences.test.ts

```diff
@@ -103,4 +104,32 @@ describe("Cell references", () => {
       expect(cellInfo.value).toBe(refValue);
     }
   });
+
+  it("should create an empty cell with styling", () => {
+    const b2 = sheet.getCellInfoAt(1, 1)!;
+    sheet.setCellStyle(
+      b2.position!.columnKey!,
+      b2.position!.rowKey!,
+      new CellStyle(new Map([["width", "50px"]])),
+    );
+
+    sheet.setCellAt(1, 1, "");
+
+    // Clearing the style should result in the cell being deleted.
+    sheet.setCellStyle(b2!.position.columnKey!, b2!.position.rowKey!, null);
+    expect(sheet.getCellInfoAt(1, 1)).toBeNull();
+  });
+
+  it("should create a cell with a reference to a non-existent cell", () => {
+    const colKey = sheet.columnPositions.get(1)!;
+    const rowKey = sheet.rowPositions.get(1)!;
+    sheet.setCell(colKey, rowKey, "");
+    const cell = sheet.setCellAt(0, 0, "=B2");
+    expect(cell.state).toBe(CellState.OK);
+    expect(sheet.getCellInfoAt(1, 1)).not.toBeNull();
+
+    // Delete the reference - this should result in the referred cell being deleted.
+    sheet.setCellAt(0, 0, "123");
+    expect(sheet.getCellInfoAt(1, 1)).toBeNull();
+  });
```

Comment:
Can we change the test to this? Previously we had to clear the cell which got me confused. This way it is shorter and less confusing.


```suggestion
  it("should create a cell with a reference to a non-existent cell", () => {
    const cell = sheet.setCellAt(0, 0, "=B5");
    expect(cell.state).toBe(CellState.OK);
    expect(sheet.getCellInfoAt(1, 4)).not.toBeNull();

    // Delete the reference - this should result in the referred cell being deleted.
    sheet.setCellAt(0, 0, "123");
    expect(sheet.getCellInfoAt(1, 4)).toBeNull();
  });
```